src/modules/commentsCounter.js

```diff
@@ -0,0 +1,6 @@
+const commentsCounter = (counter, link) => {
+  link.innerHTML = `Comments (${counter})`;
+  return counter;
+};
+
+export default commentsCounter;
```

Comment:
Good job implementing this feature in this form, but the [requirement](https://github.com/microverseinc/curriculum-javascript/blob/main/group-capstone/js_capstone.md#:~:text=Even%20if%20the%20API%20gives%20you%20these%20numbers%2C%20you%20will%20create%20a%20specific%20function%20to%20calculate%20these%20numbers%20in%20each%20page.%20These%20count%20functions%20need%20to%20be%20covered%20with%20unit%20tests%20using%20Jest.) strictly wants you to do it differently, that is to query the DOM for specific elements such as a tag, class or id attached to the tags specifically used for the comment and then make the count based on that. Consider updating this approach in order to be consistent with the requirements.

HINT: querySelectorAll can help you here since it returns an array of all the found DOM element. Also when doing this, make sure to take care of a case when there is no returned data from the DOM ie count is 0


ATTENTION: This also implies that the approach currently used for the tests will change